cinema/views.py

```diff
@@ -69,7 +75,7 @@
 ):
     queryset = Movie.objects.prefetch_related("genres", "actors")
     serializer_class = MovieSerializer
-    authentication_classes = (TokenAuthentication,)
+
```

Comment:
The permission class `IsAdminOrIfAuthenticatedReadOnly` is correctly applied to the `MovieViewSet`, ensuring that only admins can create movies while authenticated users can read them.